### DISCUSSION TITLE
Python API

### DISCUSSION THREAD

**Discussion by AlessandroOrsatti** [PaddlePaddle/PaddleSpeech#3612] [Q&A]
Hi everyone, is there a simple guide to use the asr, tts and voice conversion with the python API? thanks in advance

**Comment by abhishek887229** [comment]
Automatic Speech Recognition (ASR):
ASR is used to convert spoken language into written text.

Using Google Cloud Speech-to-Text API:
`pip install --upgrade google-cloud-speech`

Text-to-Speech (TTS):
TTS is used to convert text into spoken language.

Using Google Cloud Text-to-Speech API:
`pip install --upgrade google-cloud-texttospeech
`

Voice Conversion:
Voice conversion is the process of modifying a speaker's voice characteristics while preserving the linguistic content.

Using a simple Voice Conversion Library:
`pip install pydub
`

**Comment by mianyangnu** [comment]
Automatic Speech Recognition (ASR):
ASR is used to convert spoken language into written text.

Using Google Cloud Speech-to-Text API:
pip install --upgrade google-cloud-speech

Text-to-Speech (TTS):
TTS is used to convert text into spoken language.

Using Google Cloud Text-to-Speech API:
pip install --upgrade google-cloud-texttospeech 

Voice Conversion:
Voice conversion is the process of modifying a speaker's voice characteristics while preserving the linguistic content.

Using a simple Voice Conversion Library:
pip install pydub

**Reply by whshyaige** [reply]
Automatic Speech Recognition (ASR):
ASR is used to convert spoken language into written text.

Using Google Cloud Speech-to-Text API:
pip install --upgrade google-cloud-speech

Text-to-Speech (TTS):
TTS is used to convert text into spoken language.

Using Google Cloud Text-to-Speech API:
pip install --upgrade google-cloud-texttospeech

Voice Conversion:
Voice conversion is the process of modifying a speaker's voice characteristics while preserving the linguistic content.

Using a simple Voice Conversion Library:
pip install pydub

**Comment by Aldr1ck** [comment]
Automatic Speech Recognition (ASR):
ASR is used to convert spoken language into written text.

Using Google Cloud Speech-to-Text API:
pip install --upgrade google-cloud-speech

Text-to-Speech (TTS):
TTS is used to convert text into spoken language.

Using Google Cloud Text-to-Speech API:
pip install --upgrade google-cloud-texttospeech 

Voice Conversion:
Voice conversion is the process of modifying a speaker's voice characteristics while preserving the linguistic content.

Using a simple Voice Conversion Library:
pip install pydub
